__tests__/api.test.js

```diff
@@ -326,4 +325,58 @@ describe('API testing', () => {
             expect(msg).toBe("404 Not Found");
           });
       });
+
+    });
+    describe('GET /api/articles (queries)', () => {
+        test("200 - It should should return the articles sorted by any valid column (alphabetically or ascending)", () => {
+            return request(app)
+              .get("/api/articles?sort_by=author")
+              .expect(200)
+              .then(({ body: { articles } }) => {
+                expect(articles).toBeSortedBy("author");
+              });
+          });
+          test("200 - It should should return the articles based on the topic provided", () => {
+            return request(app)
+              .get("/api/articles?topic=mitch")
+              .expect(200)
+              .then(({ body: { articles } }) => {
+                articles.forEach((article) => {
+                  expect(article.topic).toEqual("mitch");
+                });
+              });
+          });
+          test("200 - It should should return the articles in ascending order when order query = ASC (not sensitive to lower or uppercase)", () => {
+            return request(app)
+              .get("/api/articles?order=ASC")
+              .expect(200)
+              .then(({ body: { articles } }) => {
+                expect(articles).toBeSortedBy("created_at");
+              });
+          });
```

Comment:
Please write another test where the topic exists, but there are no articles. For example: ?topic=paper. This should expect a status 200 and an empty array.